### PR TITLE
[WHIS-60] Create S3 bucket TF code

### PR DESCRIPTION
**Pull request by kostyaplis**
Create S3 bucket TF code

### CODE REVIEW

File: lib/ros/be/infra/files/terraform/aws/s3/main.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+resource "aws_s3_bucket" "this" {
+  count = length(var.s3_bucket_names)
+  bucket = var.s3_bucket_names[count.index]
+  acl    = "private"
+
+  tags = var.tags
+}
+
```
Comment (duanshiqiang) [inline lib/ros/be/infra/files/terraform/aws/s3/main.tf]: I think we don't need to have a separate aws provider section. Unlike cloudfront, there's no need to create a separate resource on a specific region.

File: lib/ros/be/infra/files/terraform/aws/s3/variables.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  type        = map(string)
+  default     = {}
+  description = "Tags for the created resources"
+}
+
+variable "aws_profile" {
+  type    = string
+  default = "default"
+}
+
```
Comment (duanshiqiang) [inline lib/ros/be/infra/files/terraform/aws/s3/variables.tf]: Same as before, no need to specify aws_profile

File: lib/ros/be/infra/files/terraform/aws/s3/main.tf (hunk 1 above)
Comment (kostyaplis) [reply lib/ros/be/infra/files/terraform/aws/s3/main.tf]: fixed

File: lib/ros/be/infra/files/terraform/aws/s3/variables.tf (hunk 2 above)
Comment (kostyaplis) [reply lib/ros/be/infra/files/terraform/aws/s3/variables.tf]: fixed

**Pull request merged**